src/Security/ExpressionLanguage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCacheBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCacheBundle\Security;
+
+use Symfony\Component\Security\Core\Authorization\ExpressionLanguage as BaseExpressionLanguage;
+
+/**
+ * Adds some function to the default Symfony Security ExpressionLanguage.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+class ExpressionLanguage extends BaseExpressionLanguage
```

Comment:
see discussion at https://github.com/FriendsOfSymfony/FOSHttpCacheBundle/pull/610#discussion_r1531560480

i don't think this should be in the FOSHttpCacheBundle, it has nothing directly to do with it.

and reading https://symfony.com/doc/current/security/expressions.html i would expect those expressions to be supported by core symfony. can we remove it?

Reply:
asked in symfony slack if there is any service we can use to get the "full" expression language of the application

Reply:
went with allowing a custom service and defaulting to the symfony/expression-language default implementation.